src/core/ext/transport/binder/utils/transport_stream_receiver_impl.h

```diff
@@ -69,12 +73,28 @@ class TransportStreamReceiverImpl : public TransportStreamReceiver {
   // messages will be re-ordered by it. In such case, the queueing approach will
   // work fine. Refer to the TODO in WireWriterImpl::ProcessTransaction() at
   // wire_reader_impl.cc for detecting and resolving out-of-order transactions.
-  std::map<StreamIdentifier, std::queue<Metadata>> pending_initial_metadata_
-      ABSL_GUARDED_BY(m_);
-  std::map<StreamIdentifier, std::queue<std::string>> pending_message_
-      ABSL_GUARDED_BY(m_);
-  std::map<StreamIdentifier, std::queue<std::pair<Metadata, int>>>
+  //
+  // TODO(waynetu): Consider replace absl::optional with absl::StatusOr to
```

Comment:
Obsolete comments.

Reply:
Tawei can you help remove them in internal repository? Because any local changes here will be overwrite by future import

Reply:
nvm let me do it now